composer/trainer/trainer.py

```diff
@@ -770,7 +777,7 @@ def __init__(
         if isinstance(precision, str):
             precision = Precision(precision)
 
-        _validate_precision(precision, self._device, deepspeed_enabled)
+        #_validate_precision(precision, self._device, deepspeed_enabled)
```

Comment:
Can we keep this line? Feel free to fix the function to make it work on TPU.

```suggestion
        _validate_precision(precision, self._device, deepspeed_enabled)
```